src/helm/benchmark/presentation/run_specs.conf

```diff
@@ -467,6 +467,35 @@ entries: [
   {description: "lsat_qa:model=all,task=assignment", priority: 3}
   {description: "lsat_qa:model=all,task=miscellaneous", priority: 3}
 
+  {description: "lextreme:subset=all,model=all", priority: 2}
```

Comment:
I think it would be better to move this to a new run_specs file e.g. `legal_run_specs.conf` cc @teetone